src/site/_includes/pattern-page.njk

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Could you get rid of this file and instead have pattern.njk extend post.njk?

Reply:
Do you mean I should introduce {% blocks %} to post.njk? It was made with includes in mind rather than extends.

Reply:
no, sorry for the confusion. I thought pattern.njk could use post.njk as its layout (removing the need for pattern-page.njk). But I see now that post.njk does the weird flattened permalink thing, so we have to have pattern-page.njk if we want nested urls.